Sources/Action.swift

```diff
@@ -11,25 +11,29 @@ import enum Result.NoError
 public final class Action<Input, Output, Error: Swift.Error> {
 	private let executeClosure: (Input) -> SignalProducer<Output, Error>
 	private let eventsObserver: Signal<Event<Output, Error>, NoError>.Observer
+	private let disabledErrorsObserver: Signal<(), NoError>.Observer
 
 	/// A signal of all events generated from applications of the Action.
 	///
 	/// In other words, this will send every `Event` from every signal generated
-	/// by each SignalProducer returned from apply().
+	/// by each SignalProducer returned from apply() except `ActionError.disabled`.
 	public let events: Signal<Event<Output, Error>, NoError>
 
 	/// A signal of all values generated from applications of the Action.
 	///
 	/// In other words, this will send every value from every signal generated
-	/// by each SignalProducer returned from apply().
+	/// by each SignalProducer returned from apply() except `ActionError.disabled`.
 	public let values: Signal<Output, NoError>
 
 	/// A signal of all errors generated from applications of the Action.
 	///
 	/// In other words, this will send errors from every signal generated by
-	/// each SignalProducer returned from apply().
+	/// each SignalProducer returned from apply() except `ActionError.disabled`.
```

Comment:
I think this should be a separate `-note: ` section. It doesn't actually makes sense here because `ActionError.disabled` never comes from the `SignalProducer` returned from `apply()`.